src/examples/pongxr/scene.js

```diff
@@ -0,0 +1,200 @@
+/*****
+WHEN CORE IS WORKING:
+- TEST IN THE SOCIAL PLACES
+- RESPONSIVE DESIGN: PHONE
+- RESPONSIVE DESIGN: MAGIC LEAP
+*/
+
+import { Scene, Object3D } from "three";
+import XRInput from "../../engine/xrinput"
+import PeerConnection from '../../engine/networking/PeerConnection'
+import State from "../../engine/state";
+import Physics from "../../engine/physics/physics"
+
+import HostBot from "../../engine/util/hostbot"
+import Level from "./level"
+import Paddle from "./paddle"
+import Ball from "./ball"
+import PlacementCube from "./placementcube"
+
+const scene = new Scene();
+const networking = new PeerConnection(scene);
+const hostBot = new HostBot(networking);
+let ball, placementCube;
+let paddles = [];
+
+// custom States & events
+State.eventHandler.registerEvent('gameover');
+const GameState = { placement: 1, play: 2 }
+
+Physics.enableDebugger(scene);
+
+const createPongLevel = (placementCube) =>
+{
+    const targetPosition = placementCube.position;
+    const targetRotation = placementCube.rotation;
+
+    State.GameState = GameState.play;
+    const curPosRot = { position: targetPosition, rotation: targetRotation }
+    const level = new Level(curPosRot);
+    scene.add(level);
+    networking.remoteSync.addLocalObject(level, { type: "level", posRot: curPosRot }, true);
+
+    // dumb hack for the other side 
```

Comment:
might be related to the fact I had to create a seemingly unnecessary `curPosRot` object, since `addLocalObject`'s `info` property was unhappy when iterated on directly. Anyway, let's discuss.